oaknational/oak-ai-lesson-assistant
# build: release

## Description

- Release candidate for 1.1.43 (copied from ai-beta).
- Due to the repo transition, the code for 1.1.43 is already in the production branch, so this only contains a change to the change log that was missed.

## How to test

1. Go to https://oak-ai-lesson-assistant-fgakcf29s.vercel-preview.thenational.academy
2. Click on \_\_\_\_\_\_
3. You should see \_\_\_\_\_\_
